docs/src/pages/customization/how-to-customize/how-to-customize.md

```diff
@@ -165,45 +165,10 @@ Please take a look at the theme's [global overrides page](/customization/theme-c
 
 Components expose [global class names](/styles/advanced/#with-material-ui-core) to enable customization with CSS.
 
-```jsx
-const GlobalCss = withStyles({
-  // @global is handled by jss-plugin-global.
-  '@global': {
-    '.MuiButton-root': {
-      fontSize: '1rem',
-    },
-  },
-})(() => null);
-
-// …
-
-<GlobalCss />;
-```
-
-If you are using the [CssBaseline](/components/css-baseline/) component to apply global resets, it can also be used to apply global styles. For instance:
-
-```jsx
-const theme = createMuiTheme({
-  components: {
-    MuiCssBaseline: {
-      styleOverrides: {
-        '@global': {
-          html: {
-            WebkitFontSmoothing: 'auto',
-          },
-        },
-      },
-    },
-  },
```

Comment:
I believe it was the only place we were officially referencing the approach of adding global CSS with the theme (the other places are application use cases).

What will be the documentation strategy? For instance, how will we document the GlobalStyles helper, or this approach?

Reply:
I removed this part as this was mostly around customizing and this component is not suppose to be used for that. We may add a standard component page documenting it. What do you think?

Reply:
I guess it depends on the perspective. If a developer is looking for customizing the `<kbd>` elements. Would it make sense to load the "How to customize" page https://next.material-ui.com/customization/how-to-customize/ and seek fo the global section?

BTW, it looks like we should update the title "Customizing components" doesn't match the title in the side nav anymore.

Reply:
I see, ok let me add the section back with content to reference how to customize primitive element 👍

Reply:
Added - https://github.com/mui-org/material-ui/pull/24176/commits/0d5579ba5f9f92a8f75f0e7810a77eaee6f7ed63